codegen/sdk-codegen/build.gradle.kts

```diff
@@ -203,13 +203,23 @@ val AwsService.modelExtrasDir: String
 
```

Comment:
Changes in this file adds staging logic for moving generated smoke tests for each service to corresponding directory under `SmokeTests/`.